src/Http/Middleware/ForbidBannedUser.php

```diff
@@ -45,11 +45,20 @@ public function __construct(Guard $auth)
     public function handle($request, Closure $next)
     {
         $user = $this->auth->user();
+        $redirect_url = config('ban.redirect_url', false);
 
         if ($user && $user instanceof BannableContract && $user->isBanned()) {
-            return redirect()->back()->withInput()->withErrors([
-                'login' => 'This account is blocked.',
-            ]);
+            if($redirect_url){
+                return redirect($redirect_url)->withInput()->withErrors([
+                    'login' => 'This account is blocked.',
+                ]);
+            }
+            else{
+                return redirect()->back()->withInput()->withErrors([
+                    'login' => 'This account is blocked.',
```

Comment:
Errors array is duplicated and could be extracted to the variable.

Reply:
Removed error array duplication